src/events/qaMention.js

```diff
@@ -1,9 +1,13 @@
 const { Events } = require('discord.js');
 const { DISCORD_CONFIG } = require('../config');
+const { pollResults } = require('../utils/poll-vote');
 
 module.exports = {
   name: Events.MessageCreate,
   async execute(message) {
+
+    if (message.embeds[0]?.data?.type === 'poll_result') await pollResults(message)  
```

Comment:
Idk if it is possible to add here an extra validation like if author of message was BOT and if embed includes the ID of the author of the PR. 

Since if it is not the BOT the author, then we shouldn't assume it is a points votation

And if it doesn't have the ID of the author of the PR, we will don't know what is the user that we should give the points

Reply:
Maybe something like that
>![image](https://github.com/user-attachments/assets/c5c6b9ae-e335-490c-87a1-5fff40eff27f)